visa_mock/base/base_mocker.py

```diff
@@ -199,7 +199,7 @@ def send(self, scpi_string: str) -> Any:
         handler = None
 
         for regex_pattern in self.__scpi_dict__:
-            search_result = re.search(regex_pattern, scpi_string)
+            search_result = re.match(regex_pattern, scpi_string, re.IGNORECASE)
```

Comment:
If you follow my suggestion below, this becomes: `search_result = regex_pattern.match(scpi_string)`